lua/arccw/shared/attachments/ud_m16_stock_moe.lua

```diff
@@ -1,12 +1,8 @@
-att.PrintName = "AMCAR MOE Stock"
+att.PrintName = "Magpul MOE Stock"
```

Comment:
Real brand should only show up with true names on. Please keep as AMCAR for consistency across M16 attachments and set true name to MAGPUL (upper case)